src/current/_includes/releases/cloud/2023-10-04.md

```diff
@@ -0,0 +1,8 @@
+## October 4, 2023
+
+<h3 id="2023-10-04-general-changes"> General changes </h3>
+
+- New CockroachDB {{ site.data.products.dedicated }} clusters on Azure can be created in additional [cloud regions](https://www.cockroachlabs.com/docs/cockroachcloud/regions?filters=dedicated#azure-regions):
+
+  - Pune (`centralindia`)
```

Comment:
We didn't add `centralindia` yet, only `southeastasia`. We should have centralindia sometime next week.